Markdown/cmm-2.0.14/system-operators/2-installation.md

```diff
@@ -691,6 +691,28 @@ agent each time the machine is booted.
 The agent is installed in a virtualenv environment. By default, the virtualenv enviroment is located
 in the `/opt/monasca-agent/` directory.
 
+> **Note:** If you require to monitor disk mount_points which are only accessible to the root,
```

Comment:
Pls. write
"accessible to root user"
instead of
"accessible to the root"

Reply:
Done